asapdiscovery-data/asapdiscovery/data/scripts/split_fragment_screen.py

```diff
@@ -142,16 +144,34 @@ def split_ligands(directory, input_file, structure_name):
         output_file = str(lig_pdb_file.stem) + "_protein.pdb"
         save_openeye_pdb(lig_protein, output_path / output_file)
         # Delete the ligand pdb since no longer useful
-        os.remove(lig_pdb_file)
+        # os.remove(lig_pdb_file)
```

Comment:
You could make this a flag (--save-intermediates). or uncomment.

Reply:
Happy to push testing to another PR, lets go ahead here in meantime?